lib/ssl/src/ssl_config.erl

```diff
@@ -1303,7 +1303,7 @@ handle_psk(UserOpts, #{versions := Versions} = Opts) ->
             PSK1 = unicode:characters_to_binary(PSK0),
             PSKSize = byte_size(PSK1),
             assert_version_dep(psk_identity, Versions, ['tlsv1','tlsv1.1','tlsv1.2']),
-            option_error(not (0 < PSKSize andalso PSKSize < 65536),
+            option_error(not (PSKSize < 65536),
```

Comment:
I think it would make sense to 0 <= PSKSize andalso PSKSize < 65536  when Role == client
for interop reasons that you have, but not allow this for the server as this I think it does not make sense from a security perspective.

Reply:
At least I think we should check that it is greater than or equal to zero.  Maybe we can allow it even if it was not allowed before as it sort of equally stupid as using only one byte, and all this is sort of legacy way to do things anyway. What do you think @dgud, @u3s ?